core/chain_util.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	blockHashPre  = []byte("block-hash-")
 	blockNumPre   = []byte("block-num-")
-	expDiffPeriod = big.NewInt(100000)
+	ExpDiffPeriod = big.NewInt(100000)
```

Comment:
https://github.com/ethereum/go-ethereum/pull/1611/files#diff-8d155028f84d7906b9c88c7630d3f7c4R156 Max int. It won't the solve the problem in 20 years anymore ... :-P